core/src/main/scala/cromwell/core/MetadataQuery.scala

```diff
@@ -1,9 +1,8 @@
-package cromwell.services.metadata
+package cromwell.core
```

Comment:
Wait how does core know about metadata stuff ?

Reply:
Oh sorry, everything it references is in this file. I'm not sure about Metadata classes being defined in core though :/
At least I would move it to a `metadata` package.

Reply:
Revert

Reply:
This is so that the conversion from X to WOM can create a list of `MetadataEvent`s to later be sent to the metadata service.

I wonder if I can change this to instead create a list of "things that will eventually become `MetadataEvent`s" and get the MWDA to "convert and send" instead of just forwarding on the prebuilt `List[MetadataEvent]`

Reply:
It would probably end up being 
```scala
case class NotMetadataEvent { copy / paste form MetadataEvent }
``` 
but it might be better than having the real Metadata classes defined in core.

Reply:
I suppose I **can** envision having key metadata concepts defined outside of `services`, and then the stuff behind `services` is just responsible for how metadata events are persisted.

However, **if** that's going to happen it should be pretty wholesale & well thought out, and not just moving a single class across because it's convenient